### ISSUE TITLE
Failed to publish npm package

### ISSUE THREAD

**Issue by AlpacaGoesCrazy** [michalkvasnicak/aws-lambda-graphql#28]
Current NPM package 0.8.0 still resolves to previous version
Reproduction: https://codesandbox.io/embed/wizardly-gould-nz0wi

**Comment by michalkvasnicak** [comment]
@AlpacaGoesCrazy 

![image](https://user-images.githubusercontent.com/174716/66320204-abac5780-e91e-11e9-8e8a-39228a422a17.png)

In your codesandbox you have `package.json` with the following content:

```json
{
  "name": "parcel-sandbox",
  "version": "1.0.0",
  "description": "Simple Parcel Sandbox",
  "main": "index.html",
  "scripts": {
    "start": "parcel index.html --open",
    "build": "parcel build index.html"
  },
  "dependencies": {
    "apollo-link": "1.2.13",
    "aws-lambda-graphql": "0.7.2",
    "aws-sdk": "2.543.0",
    "graphql": "14.5.8",
    "graphql-subscriptions": "1.1.0"
  },
  "devDependencies": {
    "@babel/core": "7.2.0",
    "parcel-bundler": "^1.6.1"
  }
}
```

**Comment by AlpacaGoesCrazy** [comment]
@michalkvasnicak changed to 0.8.0 and it still resolves to old package

**Comment by michalkvasnicak** [comment]
That's weird I can't replicate it. See https://codesandbox.io/embed/sad-mayer-yefcj

![image](https://user-images.githubusercontent.com/174716/66320917-fed2da00-e91f-11e9-869d-902034a1f7dd.png)

And

![image](https://user-images.githubusercontent.com/174716/66320937-06927e80-e920-11e9-92fa-cefcd52cced9.png)

**Comment by AlpacaGoesCrazy** [comment]
Could you look at console output?

**Comment by michalkvasnicak** [comment]
You're right, I have wrong package.json configurations so it did not run `build` on publish, because `prepublish` script is missing. Fixed in `0.8.1`. I will mark `0.8.0` as deprecated.
